### ISSUE TITLE
Mass dependence

### ISSUE THREAD

**Issue by sottcap** [anyuzx/Lammps_brownian#3]
I have a question about BD. As long as I know, Brownian Dynamics should be mass-independent. But, when I went through your codes, I found it seems your implementation is mass-dependent (See line 103 of fix_bd.cpp

```
        randf = sqrt(rmass[i]) * gfactor;
        x[i][0] += dtv * dtfm * (f[i][0]+randf*random->gaussian());
        x[i][1] += dtv * dtfm * (f[i][1]+randf*random->gaussian());
        x[i][2] += dtv * dtfm * (f[i][2]+randf*random->gaussian());
```

or 

line 112
```
        dtfm = dtf / mass[type[i]];
        randf = sqrt(mass[type[i]]) * gfactor;
        x[i][0] += dtv * dtfm * (f[i][0]+randf*random->gaussian());
        x[i][1] += dtv * dtfm * (f[i][1]+randf*random->gaussian());
        x[i][2] += dtv * dtfm * (f[i][2]+randf*random->gaussian());
```
).
Have you checked your program with masses other than 1.0? I am not sure if this will work.

**Comment by anyuzx** [comment]
you are right that bd is mass-independent, as only the friction coefficient matters. Here, as long as the value of the friction coefficient is the same, the results would be the same as well. For instance, you can use mass=1 and damp=1, which would be the same as mass=10 and damp=10. In another word, the value of the friction coefficient is not a direct parameter. Instead, it is based on mass and damp parameter.
